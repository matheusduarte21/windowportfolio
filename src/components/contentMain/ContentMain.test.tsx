import { render } from "@testing-library/react"
import { ContentMain } from "./ContentMain"

test('verificando components filhos', ()=>{
    const {container} =  render(<ContentMain/>)
    const contentMain = container.querySelector('.container-main')

    expect(contentMain?.children.length).toBe(4)
})

test('teste de snapshot', ()=>{
    const {asFragment} = render(<ContentMain/>)
    expect(asFragment).toMatchSnapshot()
})