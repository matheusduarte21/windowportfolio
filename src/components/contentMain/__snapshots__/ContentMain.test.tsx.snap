// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste de snapshot 1`] = `[Function]`;
